src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JavaScriptCapstone</title>
+</head>
+<body>
+
+    <header id="header">
+        <nav>
+            <div class="logo-container">
+                <img src="./logo.png" alt="Logo">
+            </div>
+            <ul class="menu">
+                <li><a href="#" data-category="Seafood">Seafood</a><span></span></li>
+                <li><a href="#" data-category="Lamb">Lamb</a><span></span></li>
+                <li><a href="#" data-category="Chicken">Chicken</a><span></span></li>
+                <li><a href="#" data-category="Vegetarian">Vegetarian</a><span></span></li>
+            </ul>
```

Comment:
- [ ] [OPTIONAL] You may have forgotten to add the items counter on the navigation menu.

![counters](https://github.com/MohammadYaser/Javascript-capstone/assets/53124548/363bd594-97a5-4d57-b388-77277ff9c279)